dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -0,0 +1,85 @@
+require "base64"
+
+class LtiV1Controller < ApplicationController
+  # Don't require an authenticity token because LTI Platforms POST to this
+  # controller.
+  skip_before_action :verify_authenticity_token
+
+  # [GET/POST] /lti/v1/login(/:platform_id)
+  #
+  # Most LTI Platforms should send a client_id as part of the login request.
+  # However, it is not required per the LTI 1.3 spec. In these cases, we can
+  # supply clients with a unique login URL, so we can identify the caller's
+  # identity. The requst will always contain the iss param, as required by the
+  # LTI 1.3 standard
+  # https://www.imsglobal.org/spec/security/v1p0/#step-1-third-party-initiated-login
+  def login
+    # First check if the request contains a client_id
```

Comment:
I think these inline if / elsif / else comments are overkill and would rather remove them and let the code speak for itself. I like the link to the spec above and the longer comment.

Reply:
I agree. I think the long comment above explains what is needed.